uploader.html

```diff
@@ -111,8 +107,8 @@ <h4 class="modal-title">Upload <%= filetype %></h4>
 
           <div class="modal-body">
             <div id="select-file-section" class="row"></div>
-            <div id="media-data-section" class="row col-sm-8 pull-right"></div>
             <div id="selected-media-section" class="row col-sm-12"></div>
+            <div id="media-data-section" class="row col-sm-8 pull-right"></div>
```

Comment:
Is this meant to be 8 columns wide, but right-aligned instead of left-aligned? If so, should use `col-sm-offset-4` instead of `pull-right`.